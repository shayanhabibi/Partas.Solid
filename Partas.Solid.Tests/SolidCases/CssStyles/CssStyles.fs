module Partas.Solid.Tests.SolidCases.CssStyles

open Fable.Core
open Fable.Core.JS
open Fable.Core.JsInterop
open Partas.Solid
open Partas.Solid.Style
open Partas.Solid.Experimental.U

[<SolidComponent>]
let CssPropStyleTest () =
    div().style' [
        Style.backgroundColor BackgroundColor.Slategray
        "--custom-var" ==> "Chocolate"
    ]