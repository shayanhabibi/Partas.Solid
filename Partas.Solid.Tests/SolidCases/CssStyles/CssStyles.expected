
export function CssPropStyleTest() {
    return <div style={{
            "background-color": "slategray",
            "--custom-var": "Chocolate",
        }} />;
}

